app/assets/stylesheets/preact/sidebar-widget.scss

```diff
@@ -23,7 +23,7 @@
               display: block;
               padding: 5px 0px;
               font-size: 18px;
-              color: $black;
```

Comment:
You should leave this line as a fallback for browsers that don't support CSS variables.